server/pom.xml

```diff
@@ -16,17 +16,28 @@
 
 	<properties>
 		<java.version>1.8</java.version>
+    <jsonwebtoken.version>0.8.0</jsonwebtoken.version>
+    <spring-restdocs.version> 2.0.4.RELEASE</spring-restdocs.version>
 	</properties>
 
 	<dependencies>
-		<dependency>
+
+    <!-- JSON Web Token lib -->
+    <dependency>
+      <groupId>io.jsonwebtoken</groupId>
+      <artifactId>jjwt</artifactId>
+      <version>${jsonwebtoken.version}</version>
+    </dependency>
+
+
+
+
+
+    <dependency>
```

Comment:
A lot of redundant lines are typed in `pom.xml` file.

Reply:
fixed